raiden/transfermanager.py

```diff
@@ -86,68 +128,172 @@ def transfer_async(self, amount, target, identifier=None, callback=None):
         """
 
         # Create a default identifier value
-        if not identifier:
+        if identifier is None:
```

Comment:
Please do not change this. Put it back to `not identifier`.
Check the relevant discussion here: https://github.com/raiden-network/raiden/pull/147#discussion_r82794929